pkg/ovs/utils.go

```diff
@@ -61,6 +63,33 @@ func getPhysicalNetworks(
 	)
 }
 
+func getOvsPodsNodes(
+	ctx context.Context,
+	k8sClient client.Client,
+	instance *v1beta1.OVS,
+) ([]string, error) {
+
+	var nodes []string
+	podList := &corev1.PodList{}
+	podListOpts := &client.ListOptions{
+		Namespace: instance.Namespace,
+	}
+	client.MatchingLabels{
+		"service": ServiceName,
+	}.ApplyToList(podListOpts)
+
+	if err := k8sClient.List(ctx, podList, podListOpts); err != nil {
```

Comment:
(I don't know how k8s reconciles resources well so I'm just thinking out loud here. I'm sure you have a good answer to my concerns, but I'll leave it here for in case it's a real problem)

Here, we list pods that match the label for OVS pod. I wonder:

1) does it mean that the job will be executed after the OVS pod is started? Wouldn't we want to wait / block until initial configuration is complete?
2) what if something deletes some of OVS pods just before they are listed here? Will we then not create a job for the affected node? Will we not tag the configuration job as complete then? Will then a newly spawned OVS pod operate without the job executed on the node?

Reply:
Ad. 1
yes, job will be executed after OVS pods will be created. But look at the controller - this configuration job will be created only once ovs DaemonSet will be "ready" which means that there will be as many running pods as required. 
When some pod will be deleted, and created again, this config job will be created again there.

And yes, ideally would be to make run that config job before ovn-controller container is started but problem is that You can't controll really order of the containers start in pod. You can use something called "init-container" which will be run always before other containers in Pod but in this case it's not good as we need first to have ovsdb-server container to be running and then run config container.

Ad. 2.
Again, instance will not be considered ready yet as there will be less pods than required. Once it will be started back, config job should be run.